README.md

```diff
@@ -1 +1,12 @@
-# quick-start-android-boilerplate
\ No newline at end of file
+# quick-start-android-boilerplate
+
+
+
+
+## Dependencies
+The version catalog file: `libs.versions.toml` (central location for managing all dependencies accross your project, multi-module or not)
+- It is `.toml` file which manages dependencies & versions in a more scalable, organised, and modern way https://developer.android.com/build/migrate-to-catalogs
+
```

Comment:
add suggested project structure to readme

Reply:
readding this here
> add to readme: https://stackoverflow.com/questions/16804093/rename-package-in-android-studio

Reply:
include invalidate caches and restart

Reply:
add this to another PR:

Color.kt

// If you haven't decided on a color scheme yet, you can
// use these colors as placeholders which will visualize color hierarchy.
val Grey10 = Color(0xFFFAFAFA)  // Very light grey
val Grey20 = Color(0xFFF0F0F0)  // Light grey
val Grey30 = Color(0xFFD9D9D9)  // Lighter grey
val Grey40 = Color(0xFFBFBFBF)  // Light-medium grey
val Grey50 = Color(0xFFA6A6A6)  // Medium grey
val Grey60 = Color(0xFF8C8C8C)  // Dark-medium grey
val Grey70 = Color(0xFF737373)  // Dark grey
val Grey80 = Color(0xFF595959)  // Darker grey
val Grey90 = Color(0xFF404040)  // Very dark grey
val Grey100 = Color(0xFF262626) // Almost black

Reply:
add hilt test dependencies, update to latest dependency, see daysuntil and https://dagger.dev/hilt/gradle-setup

Reply:
add  name to application in  manifest <application
        android:name=".applicationnamehere"
        
        for hilt

Reply:
update scaffold and root code